### PR TITLE
Fixes #175: Added option to select resolution for image capture

### LINKED ISSUE #175: Image resolution too low for captured images
**Actual Behaviour**

Images captured from Camera fragment has very low resolution (144x176).

**Expected Behaviour**

There should be an option to select image resolution on Camera fragment or a default resolution can be selected which approximately matches the preview size.

**Screenshots of the issue**

screenshot_20170319-002312

screenshot_20170318-220301

**Would you like to work on the issue?**

Yes, I would like to work on this issue.

### PR DESCRIPTION
**Pull request by aman95**
Fixes issue #175 

Changes: 

1. Added a spinner on Camera fragment to choose from available image resolutions.
2. A compatible resolution will be auto selected based upon the preview size.

Screenshots for the change: 

screenshot_20170319-025204

screenshot_20170319-025218

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java [hunk 1]
```diff
@@ -348,6 +357,10 @@ public void onClick(View v) {
 		});
 		Camera.Parameters parameters = preview.mCamera.getParameters();
 		parameters.setFlashMode(Camera.Parameters.FLASH_MODE_AUTO);
+
+//		Adding resolutions for image capture
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: Properly follow the code. Make the // close to the comment text.

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java [hunk 2]
```diff
@@ -810,6 +863,12 @@ public void surfaceChanged(SurfaceHolder holder, int format, int w, int h) {
 			parameters.setPreviewSize(mPreviewSize.width, mPreviewSize.height);
 		}catch(Exception e){}
 		requestLayout();
+
+//		Setting up resolution of image
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: Here too

**Comment by aman95** [comment]
Done. @Abhi2424shek please review.

**Comment by Abhi2424shek** [comment]
Merge conflicts

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @aman95 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/aman95/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by mariobehling** [comment]
@aman95 There has been no activity in this PR for a long time. Would be great to get your improvements merged, but please help to resolve the conflicts. Cheers.

**Comment by Abhi2424shek** [comment]
@aman95 this one is becoming stale, can you update yourself pls?

**Comment by mariobehling** [comment]
Closing this due to inactivity. Please re-open if you plan to work on it again.

**Pull request closed without merging**